open_spiel/games/dou_dizhu/dou_dizhu_utils_test.cc

```diff
@@ -0,0 +1,313 @@
+// Copyright 2022 DeepMind Technologies Limited
```

Comment:
File level comment:

Can you do the following: 

1) Delete all code that prints output
2) Add comments describing why values are correct
3) Instead of having multiples lines setting members of the array, set the array via an initializer list? 

I've picked out a few examples via comments.